core/controllers/blog_homepage.py

```diff
@@ -143,7 +143,8 @@ def get(self, author_username):
             raise self.PageNotFoundException(
                 Exception(
                     'User with given username does not exist'))
-        if user_settings.role not in [BLOG_ADMIN, BLOG_POST_EDITOR]:
+        if not any([role in user_settings.roles for role in [
```

Comment:
Can drop the '[' and ']' around "role ... BLOG_POST_EDITOR"

Reply:
Done!